test/dialect/postgresql/test_async_pg_py3k.py

```diff
@@ -289,3 +283,20 @@ def name_f():
         async with engine.begin() as conn:
             await conn.execute(select(1))
             assert len(cache) > 0
+
+    @async_test
+    async def test_async_creator(self, metadata, async_testing_engine):
+        import asyncpg
+
+        async def async_creator():
+            conn = await asyncpg.connect(
+                "postgresql://scott:tiger@127.0.0.1:5432/test"
```

Comment:
our CI doesn't run on localhost like this.  you should transfer the URL from config.db

Reply:
Updated to pull from `config.db.url`, thanks!